client/my-sites/checkout/checkout-thank-you/marketplace/marketplace-thank-you.tsx

```diff
@@ -92,7 +92,7 @@ const MarketplaceThankYou = ( { productSlug }: IProps ): JSX.Element => {
 	const wporgPlugin = useSelector( ( state ) => getPlugin( state, productSlug ) );
 	const isWporgPluginFetched = useSelector( ( state ) => isFetched( state, productSlug ) );
 	const siteAdminUrl = useSelector( ( state ) => getSiteAdminUrl( state, siteId ) );
-	const [ pluginIcon, setPluginIcon ] = useState( null );
+	const [ pluginIcon, setPluginIcon ] = useState( '' );
```

Comment:
in contrast to another comment I made, when supplying the initial value then `useState` won't ever set `pluginIcon` to undefined. without this initial value it _can_ be `undefined` in the type.